test/e2e/app-dir/app-static/app/mismatch-tags/page.tsx

```diff
@@ -0,0 +1,26 @@
+export const dynamic = 'force-dynamic'
+
+export default async function Page() {
+  const data = await fetch(
+    'https://next-data-api-endpoint.vercel.app/api/random',
+    {
+      cache: 'no-store',
+      next: { tags: ['thankyounext'] },
+    }
+  ).then((res) => res.text())
+
+  const data2 = await fetch(
+    'https://next-data-api-endpoint.vercel.app/api/random',
+    {
+      cache: 'no-store',
```

Comment:
We need to use revalidate or similar here or we're bypassing the data cache completely

Reply:
Oh my bad. Is it better to use `force-cache`?

Reply:
Yea that should work